### ISSUE TITLE
[BUG] Missing symlink for oc client for microshift preset

### ISSUE THREAD

**Issue by adrianriobo** [crc-org/crc#3581]
CRC pre-released version `2.16.0` is missing the symlink for oc client. 

After starting the cluster when trying to setup the oc client we got 

```
ec2-user@Mac-mini bin % eval $(crc oc-env)          
ec2-user@Mac-mini bin % which oc
oc not found
```

Checking the clients folder oc client seems missing:

```
ec2-user@Mac-mini bin % ls -al /Users/ec2-user/.crc/bin/oc/
total 0
drwxr-x---  3 ec2-user  staff   96 Apr  5 08:24 .
drwxr-x---  6 ec2-user  staff  192 Apr  5 08:24 ..
lrwxr-xr-x  1 ec2-user  staff   67 Apr  5 08:24 podman -> /Users/ec2-user/.crc/cache/crc_microshift_vfkit_4.12.9_arm64/podman
```

And checking the bundle the client is correctly there:

```
ls -al /Users/ec2-user/.crc/cache/crc_microshift_vfkit_4.12.9_arm64/
total 9312000
drwxr-xr-x  10 ec2-user  staff          320 Apr  5 08:20 .
drwxrwxr-x   4 ec2-user  staff          128 Apr  5 08:20 ..
-rw-r--r--   1 ec2-user  staff         1751 Apr  5 08:19 crc-bundle-info.json
-rw-r--r--   1 ec2-user  staff  33285996544 Apr  5 08:20 crc.img
-r--------   1 ec2-user  staff          724 Apr  5 08:20 id_ecdsa_crc
-rw-r--r--   1 ec2-user  staff     53366515 Apr  5 08:20 initramfs-5.14.0-70.50.2.el9_0.aarch64.img
-rw-r--r--   1 ec2-user  staff            0 Apr  5 08:20 kubeconfig
-rwxr-xr-x   1 ec2-user  staff     99058368 Apr  5 08:20 oc
-rwxr-xr-x   1 ec2-user  staff     35089250 Apr  5 08:20 podman
-rwxr-xr-x   1 ec2-user  staff     31901816 Apr  5 08:20 vmlinuz-5.14.0-70.50.2.el9_0.aarch64
```

**Comment by adrianriobo** [comment]
As a workaround we need to create a copy oc binary **before** executing `eval $(crc oc-env)`, we can execute:

- Mac arm64:
```
cp  $HOME/.crc/cache/crc_microshift_vfkit_4.12.9_arm64/oc  $HOME/.crc/bin/oc/oc
```
- Mac amd64:
```
cp $HOME/.crc/cache/crc_microshift_vfkit_4.12.9_amd64/oc  $HOME/.crc/bin/oc/oc
```
- Linux:
```
cp $HOME/.crc/cache/crc_microshift_libvirt_4.12.9_amd64/oc  $HOME/.crc/bin/oc/oc
```
- Windows:
```
cp /Users/$env:UserName/.crc/cache/crc_microshift_hyperv_4.12.9_amd64/oc.exe /Users/$env:UserName/.crc/bin/oc
```

**Comment by anjannath** [comment]
Fixed in #3585
